onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -2728,19 +2728,19 @@ def aten_index_put_bool(
     """index_put(Tensor self, Tensor?[] indices, Tensor values, bool accumulate=False) -> Tensor"""
 
     index = op.SequenceAt(indices, 0)  # assume indices only have 1 element
-    ind_float = op.Cast(index, to=FLOAT.dtype)
+    ind_float = op.Cast(index, to=FLOAT.dtype)  # TODO: ReduceSum() cannot accept INT input
```

Comment:
```suggestion
    # FIXME: ORT ReduceSum fails on INT input even though ONNX allows it
    ind_float = op.Cast(index, to=FLOAT.dtype
```

maybe something like